tools/utils.py

```diff
@@ -157,6 +157,11 @@ def apply(target, namespace, profile, file):
     print(check_output(f'{kubectl_cmd} apply -f {file}'))
 
 
+def patch(target, namespace, profile, file, resource_type, resource_name):
```

Comment:
why do we need patch, and not just use apply?

Reply:
Since the controller configmap already exists on the ocp cluster, we want to update it rather than overwrite.

Reply:
apply will do the job anyway, you don't have to call patch